cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/mysql/MySQLBackendImpl.java

```diff
@@ -459,7 +459,11 @@ public Connection getConnection(String dbName) throws CygnusRuntimeError, Cygnus
                 if (connection == null || !connection.isValid(0)) {
```

Comment:
CNR missing (when this gets ready, of course)

Reply:
fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1744/commits/3c473f631c069fee0e90d52e0ca5d8cccedd814d